frame/staking/src/lib.rs

```diff
@@ -2013,6 +2107,108 @@ impl<T: Trait> Module<T> {
 		Ok(())
 	}
 
+	fn do_payout_stakers(
+		validator_stash: T::AccountId,
+		era: EraIndex,
+	) -> DispatchResult {
+		/* Validate input data */
+		let current_era = CurrentEra::get().unwrap_or(0);
```

Comment:
If CurrentEra is None then this means there is no current era thus I think it should return Error::InvalidaEraToReward

Reply:
fixed